css/styles.css

```diff
@@ -0,0 +1,543 @@
+}
+
+li a i {
+  position: absolute;
+  left: 16.67%;
+  right: 16.67%;
+  top: 29.17%;
+  bottom: 70.83%;
+  color: #6070ff;
+  width: 16px;
+}
+
+.menu-bar button i {
+  height: 12px;
+  width: 18px;
+
+  /*
+  left: 3px;
+  top: 6px; */
+
+  color: #6070ff;
+}
+
+li {
+  list-style: none;
+}
+
+/* Styling anchor tags */
+a {
+  text-decoration: none;
+}
+
+li a {
+  text-decoration-style: none;
+  position: relative;
+}
+
+.section-home {
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.name-welcome {
+  color: #17284d;
+  font-style: normal;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  height: 150px;
+  width: 327px;
+  display: flex;
+  align-items: center;
+  margin-top: 128px;
+  margin-bottom: 12px;
+}
+
+.description-of-me {
+  color: #344563;
+  margin-bottom: 12px;
+}
+
+.lets-connect-statement {
+  width: 125px;
+  height: 24px;
+  color: #7f8cff;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.contact-with-social {
+  color: #7f8cff;
+}
+
+.social-networks li {
+  display: inline-block;
+}
+
+.social-network {
+  text-decoration: none;
+  padding-right: 30px;
+}
+
+li .social-network i {
+  color: grey;
+  padding-right: 10px;
+  font-size: 20px;
+  font-weight: 700;
+}
+
+.social-networks .social-network i:hover {
+  color: #6070ff;
+  box-shadow: 0.001px 8px 16px rgb(64, 83, 252, 0.24);
+  border-radius: 8px;
+}
+
+/* Works and About-me section */
+.section-works {
+  margin: 16px 16px;
+}
+
+.card-works {
+  height: 220px;
+  max-width: 100%;
+  border-radius: 8px;
+  margin-bottom: 350px;
+}
+
+.section-works h1 {
+  height: 36px;
+  width: 295px;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.frame-2 {
+  height: 32px;
+  width: 236px;
+  display: flex;
+  align-items: center;
+  gap: 12px;
+  margin-bottom: 20px;
+}
+
+.frame-2 .canopy,
+.frame-2 .back-end-dev,
+.frame-2 .year {
+  height: 16px;
+  max-width: 100%;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  display: flex;
+  align-items: center;
+}
+
+.frame-2 .canopy {
+  color: #344563;
+}
+
+.frame-2 .counter {
+  height: 8px;
+  width: 8px;
+  color: #c1c7d0;
+  display: flex;
+  align-items: center;
+}
+
+.work-detail {
+  margin-bottom: -10px;
+}
+
+.work-detail p {
+  height: 76px;
+  max-width: 100%;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+
+  /* margin-bottom: 12px; */
+}
+
+.tags {
+  height: 24px;
+  width: 203px;
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  margin-bottom: 12px;
+}
+
+.tag {
+  height: 16px;
+  width: 65px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.lets-connect-about {
+  margin-bottom: 22px;
+}
+
+.action .btn {
+  height: 48px;
+  width: 162px;
+  border-radius: 8px;
+  padding: 12px;
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  margin-bottom: 40px;
+  border: 1px solid #6070ff;
+
```

Comment:
- [ ] Kindly add a hover and pressed state to the button to match the design template. YOu can use the `:hover` and `:active` pseudo-classes
![Screenshot 2022-08-20 195928](https://user-images.githubusercontent.com/94551484/186521217-abb34f06-cb44-486e-8c09-3ed371f21828.png)
.